careers/devops.markdown

```diff
@@ -0,0 +1,33 @@
+---
+title: DevOps Engineer
+summary: The Haskell Foundation seeks a passionate DevOps professional to support GHC, related Haskell projects and tooling, and the HF itself.
+postedOn: 2021-12-28
+status: Open
+---
+
+# Job Details
+
+## Responsibilities
+
+* Maintain GitLab and CI infrastructure for GHC
+* Improve CI
+* Expand the CI infrastructure to include base the base Haskell ecosystem
+* Support other Haskell libraries and tools, including HLS (Haskell Language Server)
+* Work with the Haskell Infrastructure Team
+* Support OS packaging of GHC and tools
```

Comment:
What does this entail? Just building the binaries? Another way to ask this: how does this bullet influence how we might interview / select an applicant? Do we, for example, require the applicant to be familiar with packaging requirements for all three major OSes?

Reply:
i hope not. that's something that as it becomes necessary can be learned